arangodb-net-standard/Serialization/DictionaryValueConverter.cs

```diff
@@ -26,9 +29,33 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
 
         public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
         {
-            // Use our local serializer for writing instead of the passed-in serializer
-            _serializer.Serialize(writer, value);
+            // Use a local serializer for writing instead of the passed-in serializer
+            JsonSerializer mySerializer;
+            if (_serializationOptions != null && _serializationOptions.CamelCasePropertyNamesOfObjectValuesInDictionaries)
```

Comment:
Does it mean that the option `CamelCasePropertyNamesOfObjectValuesInDictionaries` controls more than just the camel casing?
`MissingMemberHandling`, `NullValueHandling` and `UseStringEnumConversion` are also applied below when `CamelCasePropertyNamesOfObjectValuesInDictionaries` is true.

In that case, should we rename `CamelCasePropertyNamesOfObjectValuesInDictionaries` to something like: `ApplySerializationOptionsToObjectValuesInDictionaries`?

Reply:
Very good point. I've renamed it accordingly.